pkg/asset/installconfig/azure/mock/azureclient_generated.go

```diff
@@ -12,7 +12,7 @@ import (
 	subscriptions "github.com/Azure/azure-sdk-for-go/profiles/2018-03-01/resources/mgmt/subscriptions"
 	network "github.com/Azure/azure-sdk-for-go/profiles/2020-09-01/network/mgmt/network"
 	compute "github.com/Azure/azure-sdk-for-go/profiles/latest/compute/mgmt/compute"
-	gomock "go.uber.org/mock/gomock"
+	gomock "github.com/golang/mock/gomock"
```

Comment:
It looks like this change is causing issues. 

It should stick with the go.uber.org/mock package as that is the new standard. Not sure if that was updated by some automation?

Reply:
Everytime I run go generate mockgen, this creeps up.

Reply:
Oh, I haven't hit that. If you don't want to look into it now, open a bug with the reproducer and we'll fix it.